### PR TITLE
feat(curriculum): add English Morning/Evening Routine block

### PR DESCRIPTION
**Pull request by nieldakarla**
Notes for reviewers:

Task 41, 53: quotation marks were left on purpose on specific parts of the description.

Tasks to keep the the backticks in the answers:
92, 93

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Ref: #52400

### CODE REVIEW

**Review by moT01 — COMMENTED** [review]
This looks real good @nieldakarla 🎉 

I didn't look over it too hard, mainly just went through it to make sure everything is working. The one main thing I didn't like is with the descriptions. Many of them could be shortened/condensed I think. They often have many instances of one or two sentences on their own line. I think they many times they could be combined into a few short paragraphs instead of something like 5 sentences, each on their own line. It looks a little weird in my opinion. I left some comments on a few of the challenges about this. There's a few that use quite a few examples, as well - maybe just a preference of mine, but I would try to stick with one or two examples to try and keep it shorter.

A few other things that I could mention:
- I noticed one challenge that had the examples in a bulleted list. I don't think any others did. I would try to keep things consistent and maybe not use that list.
- Some of the feedback is quite generic. e.g. "That's not right." This isn't really a problem, we default to a generic response like that if feedback isn't provided.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/65569ebbc9ef9f7b5b99827b.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+For example, `You look great!` is about appearance, not looking at a place. 
+
+It's important to be careful when commenting on someone's appearance, as it can be sensitive. 
+
+Use such comments only with people you know well and in an appropriate context. 
+
+Other examples include `You look tired`, or `You look excited`.
+
+Listen to the audio and fill in the blank with the correct word that completes the sentence.
```
[the remark this answers is not in the document]
Comment (naomi-lgbt) [reply curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/65569ebbc9ef9f7b5b99827b.md:21]: We discussed in the meeting to include this in each fill in the blank challenge, I thought? I've been adding:

```md
# --instructions--

Listen to the audio to complete the sentence below.
```

[the remark this answers is not in the document]
Comment (moT01) [reply curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/65569ebbc9ef9f7b5b99827b.md:21]: That's fine, I would put it in the `# --instructions--` header then.

[the remark this answers is not in the document]
Comment (nieldakarla) [reply curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/65569ebbc9ef9f7b5b99827b.md:21]: For challenges that have actual content in the description I'll just delete this instruction.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/6557995f0d97e1b2837a3081.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+id: 6557995f0d97e1b2837a3081
+title: Task 16
+challengeType: 19
+dashedName: task-16
+---
+
+# --description--
+
+Remember: adjectives are words that describe or give more information about nouns. They can tell you how something looks, feels, or acts. For example:
```
[the remark this answers is not in the document]
Comment (nieldakarla) [reply curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/6557995f0d97e1b2837a3081.md]: @moT01 Thank you for the review! =D 

I think I've committed the changes you suggested. When you can, please have a look.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/6557d782bc7613c06aa7dafd.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+<!--
+AUDIO REFERENCE: 
+James: An early coffee and then breakfast! I never thought of that!
+
+-->
+# --description--
+
+The word `never` is an adverb used to mean "not at any time" or "not ever". It's often used to talk about things that someone doesn't do or hasn't done. For example:
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/6557d782bc7613c06aa7dafd.md:15]:
```suggestion
The word `never` is an adverb used to mean `not at any time` or `not ever`. It's often used to talk about things that someone doesn't do or hasn't done. For example:
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/6557dc1aa6be93c1eb4e9efb.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+What does Sarah imply by using `if` in her statement about checking emails and messages?
+
+## --answers--
+
+She always finds urgent updates from her team
+
+### --feedback--
+
+`if` doesn't suggest certainty of finding urgent updates.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/6557dc1aa6be93c1eb4e9efb.md:36]:
```suggestion
`If` doesn't suggest certainty of finding urgent updates.
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/6557e026c053f9c362a1983e.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+### --feedback--
+
+Use the gerund of the verb `to be`
+
+---
+`coming`
+
+### --feedback--
+
+Use the gerund of the ver `to come`
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/6557e026c053f9c362a1983e.md:37]:
```suggestion
Use the gerund of the verb `to come`
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655a21f5512be1f7dbfe4cad.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+id: 655a21f5512be1f7dbfe4cad
+title: Task 36
+challengeType: 19
+dashedName: task-36
+---
+# --description--
+
+Words ending in `y` often describe a state or quality. For example:
+
+If you add `y`to the noun `sleep`, You create the adjective `sleepy` which means feeling a need to sleep or being tired.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655a21f5512be1f7dbfe4cad.md:11]:
```suggestion
If you add `y` to the noun `sleep`, you create the adjective `sleepy` which means feeling a need to sleep or being tired. 
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655a21f5512be1f7dbfe4cad.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+# --description--
+
+Words ending in `y` often describe a state or quality. For example:
+
+If you add `y`to the noun `sleep`, You create the adjective `sleepy` which means feeling a need to sleep or being tired. 
+
+The `y` transforms the noun `sleep` into an adjective describing a condition. Similarly:
+
+`snow`+ `y` = `snowy` (having a lot of snow)
+`Rain`+ `y` = `rainy` (having a lot of rain)
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655a21f5512be1f7dbfe4cad.md:16]:
```suggestion
`snow` + `y` = `snowy` (having a lot of snow)
`rain` + `y` = `rainy` (having a lot of rain)
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655a3132c16520fe6bf8f9be.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
+---
+id: 655a3132c16520fe6bf8f9be
+title: Task 41
+challengeType: 19
+dashedName: task-41
+---
+
+# --description--
+
+`Multiple times` means "many times" or on "several occasions". It's used to indicate that an action or event happens repeatedly. For example:
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655a3132c16520fe6bf8f9be.md:10]:
```suggestion
`Multiple times` means many times or on several occasions. It's used to indicate that an action or event happens repeatedly. For example:
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655a367638e3f401eb0235ea.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+
+`Feel groggy` is an informal term that means to feel weak, tired, or not fully awake, often as a result of poor sleep or waking up suddenly. 
+For example:
+
+`I feel groggy if I don't get enough sleep` (describes feeling tired due to lack of sleep)
+
+`Sluggish` is an adjective that describes a feeling of low energy, slow movement, or a lack of alertness. It's often used when someone feels tired, lethargic, or not able to think or move quickly. This term can apply to both physical and mental states. Here are some examples:
+
+`The computer was running sluggishly, taking a long time to respond` (describes slow performance, used metaphorically for an inanimate object)
+
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655a367638e3f401eb0235ea.md:18]:
```suggestion
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655a4e0c1176be0e1b785c96.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+---
+
+<!--
+AUDIO REFERENCE: 
+Expert: Yes, don't start your day checking your phone or emails immediately.
+-->
+
+# --description--
+
+The adverb `immediately` means "without delay" or "right away". It's used to emphasize the urgency or the need to do something at once. For example:
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655a4e0c1176be0e1b785c96.md:16]:
```suggestion
The adverb `immediately` means without delay or right away. It's used to emphasize the urgency or the need to do something at once. For example:
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655a8b62e5681235c3fb5492.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+---
+
+<!--
+AUDIO REFERENCE: 
+Jake: Good idea! After dinner, I usually help them with their homework.
+-->
+
+# --description--
+
+Let's review the usage of `them`and `their`. `Them` is used to refer to multiple people previously mentioned, and `their` shows possession or belonging to these people. For example:
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655a8b62e5681235c3fb5492.md:16]:
```suggestion
Let's review the usage of `them` and `their`. `Them` is used to refer to multiple people previously mentioned, and `their` shows possession or belonging to these people. For example:
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655a8b62e5681235c3fb5492.md [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+
+`I saw the kids. I gave them their toys.` (‘them’ refers to the kids; ‘their’ shows the toys belong to the kids)
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655a8b62e5681235c3fb5492.md:18]:
```suggestion
`I saw the kids. I gave them their toys.` (`them` refers to the kids; `their` shows the toys belong to the kids)
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655b258e8cd2985ed8412275.md [hunk 13]
```diff
@@ -0,0 +1,54 @@
+---
+id: 655b258e8cd2985ed8412275
+title: "Task 79"
+challengeType: 19
+dashedName: marias-evening-activities
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655b258e8cd2985ed8412275.md:5]:
```suggestion
dashedName: task-79
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655b283d10fee46040e0a893.md [hunk 14]
```diff
@@ -0,0 +1,54 @@
...
+---
+
+<!--
+AUDIO REFERENCE: 
+Maria: On Tuesdays, I learn Japanese. I take online lessons at 7 PM. I think it is the only day I am home so early.
+-->
+
+# --description--
+
+Listen to the audio and asnwer the question.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655b283d10fee46040e0a893.md:16]:
```suggestion
Listen to the audio and answer the question.
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655b2919ff561b60fcde19ae.md [hunk 15]
```diff
@@ -0,0 +1,54 @@
...
+
+# --description--
+
+This challenge focuses on understanding `where` Maria goes and `what` she does on Wednesdays.
+
+# --question--
+
+## --text--
+
+Where does Maria go and what she does on Wednesdays?
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655b2919ff561b60fcde19ae.md:22]:
```suggestion
Where does Maria go and what does she do on Wednesdays?
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655b2919ff561b60fcde19ae.md [hunk 16]
```diff
@@ -0,0 +1,54 @@
...
+
+## --answers--
+
+She attends a painting class at a local studio
+
+### --feedback--
+
+That is not the activity Mariaattenrs on Wednesdays.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655b2919ff561b60fcde19ae.md:30]:
```suggestion
That is not the activity Maria attends on Wednesdays.
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655b2919ff561b60fcde19ae.md [hunk 17]
```diff
@@ -0,0 +1,54 @@
...
+
+She goes to a local theater group for practice and performs in small plays
+
+---
+
+She visits a friend's house to watch movies
+
+### --feedback--
+
+That is not the activity Mariaattenrs on Wednesdays.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655b2919ff561b60fcde19ae.md:42]:
```suggestion
That is not the activity Maria attends on Wednesdays.
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655b2919ff561b60fcde19ae.md [hunk 18]
```diff
@@ -0,0 +1,54 @@
...
+
+---
+
+She works late at her office
+
+### --feedback--
+
+That is not the activity Mariaattenrs on Wednesdays.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655b2919ff561b60fcde19ae.md:50]:
```suggestion
That is not the activity Maria attends on Wednesdays.
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655b32b2812874680f3198d3.md [hunk 19]
```diff
@@ -0,0 +1,46 @@
...
+---
+
+<!--
+AUDIO REFERENCE: 
+Maria: Sure, Brian! Just don't forget to give yourself time to rest. Find what your interests are and have some fun!
+-->
+
+# --description--
+
+Listen to the audio and complente the sentence
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655b32b2812874680f3198d3.md:16]:
```suggestion
Listen to the audio and complete the sentence.
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655b38c1f5351d6c827c8e8f.md [hunk 20]
```diff
@@ -0,0 +1,54 @@
...
+
+What activities does Brian do on Sundays?
+
+## --answers--
+
+Walk in the park in the morning and works on coding in the afternoon
+
+### --feedback--
+
+Ther time of the day is wrong.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-your-morning-or-evening-routine/655b38c1f5351d6c827c8e8f.md:30]:
```suggestion
The time of the day is wrong.
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Review by naomi-lgbt — APPROVED** [review]

**Review by DanielRosa74 — APPROVED** [review]
LGTM, overall.

**Pull request merged**